src/linagora.esn.unifiedinbox/app/directives/subheaders.js

```diff
@@ -39,13 +39,6 @@ angular.module('linagora.esn.unifiedinbox')
     };
   })
 
-  .directive('fullscreenEditFormSubheader', function() {
```

Comment:
Why is that removed ?

Reply:
Because it is not used anymore. And the template has been not existed. This belongs to the commit that I explained.